tests/unittests/flows/llm_flows/test_functions_request_euc.py

```diff
@@ -347,26 +347,6 @@ def call_external_api2(tool_context: ToolContext) -> int:
 
 
 def test_function_get_auth_response_partial():
```

Comment:
I was the original author of these test cases, so can vouch for this re-write staying in the spirit of the original test intention.